test/system/160-volumes.bats

```diff
@@ -125,13 +126,17 @@ EOF
         expect_msg='.* exec user process caused.*permission denied'
     fi
 
-    run_podman ${expect_rc} run --rm --volume $myvolume:/vol:z $IMAGE /vol/myscript
+    run_podman ${expect_rc} run --rm --volume $myvolume:/vol:noexec,z $IMAGE /vol/myscript
     is "$output" "$expect_msg" "run on volume, noexec"
 
     # With exec, it should pass
     run_podman run --rm -v $myvolume:/vol:z,exec $IMAGE /vol/myscript
     is "$output" "got here -$rand-" "script in volume is runnable with exec"
 
```

Comment:
remove these

Reply:
Sorry for brusqueness, I just wasted too much time typing, then having my comments lost because of outdated diffs, then typing again, then losing them AGAIN DAMN YOU GITHUB. Didn't want to do that again.

Anyhow, my point was, system tests aren't intended to cover every possible situation - they're just a last-minute check for basic functionality. Testing the default is a good enough check for exec; testing noexec, likewise. We don't IMHO need a full test just for parsing 'exec'.

Reply:
Fixed, thanks